keras/backend/mxnet_backend.py

```diff
@@ -228,83 +207,53 @@ def __rmul__(self, other):
     #                 lhs=self.symbol,
     #                 rhs=other.symbol), self, other)
 
-    # def __gt__(self, other):
-    #     if isinstance(other, Number):
-    #         return KerasSymbol(self.symbol > other)
-    #     else:
-    #         return KerasSymbol(
-    #             mx.sym.broadcast_greater(
-    #                 lhs=self.symbol,
-    #                 rhs=other.symbol))
-
-    # def __ge__(self, other):
-    #     if isinstance(other, Number):
-    #         return KerasSymbol(self.symbol >= other)
-    #     else:
-    #         return KerasSymbol(
-    #             mx.sym.broadcast_greater_equal(
-    #                 lhs=self.symbol,
-    #                 rhs=other.symbol))
-
-    # def __lt__(self, other):
-    #     if isinstance(other, Number):
-    #         return KerasSymbol(
-    #             self.symbol < other)
-    #     else:
-    #         return KerasSymbol(
-    #             mx.sym.broadcast_lesser(
-    #                 lhs=self.symbol,
-    #                 rhs=other.symbol))
-
-    # def __le__(self, other):
-    #     if isinstance(other, Number):
-    #         return KerasSymbol(
-    #             self.symbol <= other)
-    #     else:
-    #         return KerasSymbol(
-    #             mx.sym.broadcast_lesser_equal(
-    #                 lhs=self.symbol,
-    #                 rhs=other.symbol))
-
-    # def __gt__(self, other):
-    #     if isinstance(other, Number):
-    #         return KerasSymbol(
-    #             self.symbol > other)
-    #     else:
-    #         return KerasSymbol(
-    #             mx.sym.broadcast_greater(
-    #                 lhs=self.symbol,
-    #                 rhs=other.symbol))
-
-    # def __ge__(self, other):
-    #     if isinstance(other, Number):
-    #         return KerasSymbol(
-    #             self.symbol >= other)
-    #     else:
-    #         return KerasSymbol(
-    #             mx.sym.broadcast_greater_equal(
-    #                 lhs=self.symbol,
-    #                 rhs=other.symbol))
-
-    # def __lt__(self, other):
-    #     if isinstance(other, Number):
-    #         return KerasSymbol(
-    #             self.symbol < other)
-    #     else:
-    #         return KerasSymbol(
-    #             mx.sym.broadcast_lesser(
-    #                 lhs=self.symbol,
-    #                 rhs=other.symbol))
-
-    # def __le__(self, other):
-    #     if isinstance(other, Number):
-    #         return KerasSymbol(
-    #             self.symbol <= other)
-    #     else:
-    #         return KerasSymbol(
-    #             mx.sym.broadcast_lesser_equal(
-    #                 lhs=self.symbol,
-    #                 rhs=other.symbol))
+    def __gt__(self, other):
```

Comment:
where are these used? tf doesn't overload these so Keras frontend shouldn't rely on it